hardware/ucm/standard.md

```diff
@@ -4,6 +4,23 @@
 
 The UCM Standard defines a consistent hardware and software interface for the most widely used microcontroller peripherals.  Universal Compute Modules are based on the 200 pin SO-DIMM form factor.  They adhere to a standard pinout across models making it easy to change modules to adapt to the needs of your product.  
 
+## The SO-DIMM Socket
+
+![200 pin DDR2 SO-DIMM socket](images/200-pin-ddr2-so-dimm.jpg)
+
+All of our UCMs use the same 200 pin SO-DIMM socket that was originally made for DDR2 memory modules. The fastest way to get started with UCMs is by using the optional boards and displays described on the [UCM Development Options](development-options.md) page. You can also incorporate any of our UCMs into your own custom design by adding the correct SO-DIMM socket to your circuit board.
+
+> [!Tip]
+> If your custom design uses or might use Wi-Fi, it's a good idea to expose the pins needed to update the Wi-Fi firmware. These pins are Wi-Fi BOOT0 (SO-DIMM pin 108), Wi-Fi TXD (SO-DIMM pin 93), Wi-Fi RXD (SO-DIMM pin 94), and Wi-Fi GPIO 0 (SO-DIMM pin 36).
```

Comment:
This is device specific. Change the tip to be along the lines of "make sure to expose required pins in your design. See device specifications for details.".

See 8.1 in https://old.ghielectronics.com/downloads/man/G120_and_G120E_Datasheet.pdf for an example of what we did in NETMF with the G120